script/_dhcpv6_pd_ref

```diff
@@ -30,39 +30,122 @@
 #       This script manipulates DHCPv6-PD-REF configuration.
 #
 
-customise_dhcpcd_conf()
+DHCPCD_ENTER_HOOK="/etc/dhcpcd.enter-hook"
+DHCPCD_EXIT_HOOK="/etc/dhcpcd.exit-hook"
+
+PD_DAEMON_DIR="/opt/pd-daemon"
+PD_DAEMON_PATH="${PD_DAEMON_DIR}/dhcp6_pd_daemon.py"
+PD_DAEMON_SERVICE_NAME="dhcp6_pd_daemon.service"
+PD_DAEMON_SERVICE_PATH="/etc/systemd/system/${PD_DAEMON_SERVICE_NAME}"
+
+DHCP_CONFIG_PATH="/etc/dhcpcd.conf"
+DHCP_CONFIG_ORIG_PATH="/etc/dhcpcd.conf.orig"
+DHCP_CONFIG_PD_PATH="/etc/dhcpcd.conf.pd"
+DHCP_CONFIG_NO_PD_PATH="/etc/dhcpcd.conf.no-pd"
+
+# Create dhcpcd configuration file with ipv6 prefix request.
+create_dhcpcd_conf_pd()
 {
-    # This has to be run after script/_border_routing, and this will
-    # invalidate all changes to dhcpcd.conf made by script/_border_routing.
-    sudo tee /etc/dhcpcd.conf >/dev/null <<EOF
+    sudo tee ${DHCP_CONFIG_PD_PATH} >/dev/null <<EOF
 noipv6rs # disable router solicitation
 interface eth0
```

Comment:
Should we use `eth0`, or `INFRA_IF` or define another variable for the upstream interface?

Reply:
Ideally we could set the upstream interface according to the environment variables `INFRA_IF ` of `script/setup`.

Reply:
I mean should we replace the `eth0` here by some variables?

Reply:
I see, will update